### ISSUE TITLE
usbip error

### ISSUE THREAD

**Issue by wuqijian-site** [dorssel/usbipd-win#674]
I ran 'usbipd wsl list' on Win10 and got the following error:

Unhandled exception. System.AggregateException: One or more errors occurred. (Unrecognized line format: '{'.)
 ---> System.FormatException: Unrecognized line format: '{'.
   at Microsoft.Extensions.Configuration.Ini.IniStreamConfigurationProvider.Read(Stream)
   at Usbipd.WslDistributions.GetVMSwitchName() in D:\a\usbipd-win\usbipd-win\Usbipd\WslDistributions.cs:line 66
   at Usbipd.WslDistributions.<>c.<CreateAsync>b__25_0(NetworkInterface) in D:\a\usbipd-win\usbipd-win\Usbipd\WslDistributions.cs:line 104
   at System.Linq.Enumerable.TryGetFirst[TSource](IEnumerable`1, Func`2, Boolean& )
   at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1, Func`2)
   at Usbipd.WslDistributions.CreateAsync(CancellationToken) in D:\a\usbipd-win\usbipd-win\Usbipd\WslDistributions.cs:line 103
   at Usbipd.CommandHandlers.GetDistributionsAsync(IConsole, CancellationToken) in D:\a\usbipd-win\usbipd-win\Usbipd\CommandHandlers.cs:line 475
   at Usbipd.CommandHandlers.Usbipd.ICommandHandlers.WslList(Boolean, IConsole, CancellationToken) in D:\a\usbipd-win\usbipd-win\Usbipd\CommandHandlers.cs:line 808
   at Usbipd.Program.<>c__DisplayClass11_8.<<Run>b__57>d.MoveNext() in D:\a\usbipd-win\usbipd-win\Usbipd\Program.cs:line 606
--- End of stack trace from previous location ---
   at System.CommandLine.Invocation.AnonymousCommandHandler.InvokeAsync(InvocationContext)
   at System.CommandLine.Invocation.InvocationPipeline.<>c__DisplayClass4_0.<<BuildInvocationChain>b__0>d.MoveNext()
--- End of stack trace from previous location ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass17_0.<<UseParseErrorReporting>b__0>d.MoveNext()
--- End of stack trace from previous location ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass12_0.<<UseHelp>b__0>d.MoveNext()
--- End of stack trace from previous location ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass22_0.<<UseVersionOption>b__0>d.MoveNext()--- End of stack trace from previous location ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass19_0.<<UseTypoCorrections>b__0>d.MoveNext()
--- End of stack trace from previous location ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c.<<UseSuggestDirective>b__18_0>d.MoveNext()
--- End of stack trace from previous location ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass16_0.<<UseParseDirective>b__0>d.MoveNext()
--- End of stack trace from previous location ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c.<<RegisterWithDotnetSuggest>b__5_0>d.MoveNext()
--- End of stack trace from previous location ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c.<<CancelOnProcessTermination>b__1_0>d.MoveNext()
--- End of stack trace from previous location ---
   at System.CommandLine.Invocation.InvocationPipeline.<InvokeAsync>g__FullInvocationChainAsync|2_0(InvocationContext)
   at System.CommandLine.Parsing.ParseResultExtensions.InvokeAsync(ParseResult, IConsole )
   at System.CommandLine.Parsing.ParserExtensions.InvokeAsync(Parser, String[], IConsole )
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean)
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean)
   at System.Threading.Tasks.Task`1.get_Result()
   at Usbipd.Program.Run(IConsole , ICommandHandlers, String[] ) in D:\a\usbipd-win\usbipd-win\Usbipd\Program.cs:line 664
   at Usbipd.Program.Main(String[]) in D:\a\usbipd-win\usbipd-win\Usbipd\Program.cs:line 120

OS Version:  10.0.19045 N/A Build 19045

**Comment by wuqijian-site** [comment]
Oh, my configuration file is wrong.thank you.

**Comment by wuqijian-site** [comment]
When I cleared the .wslconfig file and executed again, I encountered an error. Here are my command and the output:

PS C:\> usbipd wsl list
BUSID  VID:PID    DEVICE                                                        STATE
1-3    1a2c:2124  USB 输入设备                                                  Not attached
2-1    0000:3825  USB 输入设备                                                  Not attached
2-2    0a12:0001  Generic Bluetooth Radio                                       Not attached

usbipd: warning: USB filter 'UsbDk' is known to be incompatible with this software; 'bind --force' will be required.
PS C:\> usbipd wsl attach --busid 2-2
usbipd: warning: USB filter 'UsbDk' is known to be incompatible with this software; 'bind --force' will be required.
<3>WSL (1036) ERROR: UtilTranslatePathList:2803: Failed to translate F:\Bandizip\
usbip: error: Attach Request for 2-2 failed - Device in error state

usbipd: error: Failed to attach device with busid '2-2'.
PS C:\> usbipd wsl attach --busid=2-2
usbipd: warning: USB filter 'UsbDk' is known to be incompatible with this software; 'bind --force' will be required.
<3>WSL (1075) ERROR: UtilTranslatePathList:2803: Failed to translate F:\Bandizip\
usbip: error: Attach Request for 2-2 failed - Device in error state

I don’t understand why it is trying to access the path F:\Bandizip, but I can assure you that I don’t have an F drive.
